contrib/inventory/stacki.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2016, Hugh Ma <hugh.ma@flextronics.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+# Stacki inventory script
+# Configure stacki.yml with proper auth information and place in the following:
+#  - ../inventory/stacki.yml
+#  - /etc/stacki/stacki.yaml
```

Comment:
I might be missing some significance, but should these file extensions be the same?  .yaml vs .yml?